src/Params/AbstractListParams.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Avtocod\B2BApi\Params;
+
+abstract class AbstractListParams
+{
+    /**
+     * Description of query for fetching list data.
+     *
+     * @var string
+     */
+    private $query = '_all';
+
+    /**
+     * Page number.
+     *
+     * @var int
+     */
+    private $page = 1;
+
+    /**
+     * Items per page.
+     *
+     * @var int
+     */
+    private $per_page = 20;
+
+    /**
+     * Pagination offset.
+     *
+     * @var int
+     */
+    private $offset = 0;
+
+    /**
+     * Sorting rules.
+     *
+     * @var string
+     */
+    private $sort_by = '-created_at';
+
+    /**
+     * True, if necessary include list item content into response.
+     *
+     * @var bool
+     */
+    private $include_content = false;
```

Comment:
In my opinion `with_content` is better than `include_content`

Reply:
Yes I agree 0a090fa70454885ab612e83625b4c306d32e3652

Reply:
great!